### PR TITLE
Update CI configs to v0.4.7

### PR DESCRIPTION
**Pull request by pionbot**
Update lint scripts and CI configs.

### CODE REVIEW

**Comment by scorpionknifes** [comment]
I'm taking this one.

**Comment by scorpionknifes** [comment]
I'm looking at ```chunk_abort.go``` and ```chunk_error.go```, these two files seems to be the same (dupl). I'm wondering if there is any way to define the two struct without dupl. Maybe this is something we can fix this after generics comes out in golang.

File: chunk_abort.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package sctp
+package sctp // nolint:dupl
```
Comment (scorpionknifes) [inline chunk_abort.go:1]: duplicate of chuck_error.go

File: chunk_error.go [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package sctp
+package sctp // nolint:dupl
```
Comment (scorpionknifes) [inline chunk_error.go:1]: duplicate of chuck_abort.go

File: chunk_forward_tsn.go [hunk 3]
```diff
@@ -133,7 +135,7 @@ func (s *chunkForwardTSNStream) unmarshal(raw []byte) error {
 	return nil
 }
 
-func (s *chunkForwardTSNStream) marshal() ([]byte, error) {
+func (s *chunkForwardTSNStream) marshal() ([]byte, error) { // nolint:unparam
```
Comment (scorpionknifes) [inline chunk_forward_tsn.go:138]: Keeping param to keep style to unmarshal(). don't know if the error param would be ever be used?

File: chunk_init.go [hunk 4]
```diff
@@ -1,4 +1,4 @@
-package sctp
+package sctp // nolint:dupl
```
Comment (scorpionknifes) [inline chunk_init.go:1]: duplicate of chuck_init_ack.go

File: chunk_init_ack.go [hunk 5]
```diff
@@ -1,4 +1,4 @@
-package sctp
+package sctp // nolint:dupl
```
Comment (scorpionknifes) [inline chunk_init_ack.go:1]: duplicate of chuck_init.go

**Review by scorpionknifes — COMMENTED** [review]
duplicated struct/files and one unsure lint.

File: examples/ping-pong/ping.go [hunk 6]
```diff
@@ -25,14 +25,14 @@ func main() {
...
 	defer a.Close()
 	fmt.Println("created a client")
 
 	stream, err := a.OpenStream(0, sctp.PayloadTypeWebRTCString)
 	if err != nil {
-		log.Fatal(err)
+		log.Panic(err)
 	}
 	defer stream.Close()
 	fmt.Println("opened a stream")
```
Comment (scorpionknifes) [inline examples/ping-pong/ping.go:38]: Got ```exitAfterDefer``` from ```gocritic```
I read about https://quasilyte.dev/blog/post/log-fatal-vs-log-panic/
Decided to change ```log.Fatal()``` to ```log.Panic()```

**Review by scorpionknifes — COMMENTED** [review]
Forgot to add this. This effects ```ping.go``` and ```pong.go``` in ```examples/ping-pong```

**Review by Sean-Der — APPROVED** [review]

**Pull request merged**